source/patterns/@aws-solutions-constructs/aws-lambda-opensearch/test/integ.no-arguments.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ *  Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+/// !cdk-integ *
+import { App, Stack } from "aws-cdk-lib";
+import { LambdaToOpenSearch } from "../lib";
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import { generateIntegStackName } from '@aws-solutions-constructs/core';
+
+// Setup
+const app = new App();
+const stack = new Stack(app, generateIntegStackName(__filename));
+
+const lambdaProps: lambda.FunctionProps = {
+  code: lambda.Code.fromAsset(`${__dirname}/lambda`),
+  runtime: lambda.Runtime.NODEJS_14_X,
+  handler: 'index.handler'
+};
+
+const openSearchDomain = 'solutions-constructs-domain';
```

Comment:
Any reason why we don't specify a cognito domain like we do in the Elasticsearch version of this test?

Reply:
In the `integ.no-arguments.ts` for `aws-lambda-opensearch`, I used only the required arguments to deploy the stack. Specifying both OpenSearch and Cognito domain separately is handled in `integ.domain-arguments.ts`.